mailbox/store/src/main/java/org/apache/james/mailbox/store/MessageResultImpl.java

```diff
@@ -232,7 +233,7 @@ public Content getBody() {
         }
 
         public void setBody(Content body) {
-            content = content | FetchGroup.BODY_CONTENT;
+            content = content | FetchGroup.BODY_CONTENT_MASK;
```

Comment:
FetchGroup.FULL_CONTENT.or(content) ?

Reply:
This is not stronlgly typed. Won't adress it here.

Let's keep some stuff for upcoming refactorings...